pdns/receiver.cc

```diff
@@ -566,8 +566,13 @@ int main(int argc, char **argv)
       
       if(::arg().mustDo("control-console"))
         dl=make_unique<DynListener>();
-      else
-        dl=std::unique_ptr<DynListener>(new DynListener(s_programname));
+      else {
+        try {
+           dl=std::unique_ptr<DynListener>(new DynListener(s_programname));
+        } catch (boost::exception_detail::clone_impl<boost::exception_detail::error_info_injector<boost::io::too_few_args> >& e) {
```

Comment:
any less specific exception we can catch?

Reply:
where is this exception coming from, anyway? :)

Reply:
it's what coverity complained, so i copied it from there.

Reply:
let's remove it

Reply:
lets remove it indeed, the source for this is impossible

```
throw runtime_error((boost::format("Error: Unable to parse group ID %s") % cstr).str() );
```